src/sql/parts/notebook/outputs/style/index.css

```diff
@@ -0,0 +1,467 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+/*-----------------------------------------------------------------------------
+| RenderedText
+|----------------------------------------------------------------------------*/
+
+.jp-RenderedText {
```

Comment:
OK... so for this file, I think we need to be very careful about global imports. Please check with @anthonydresser about this. I suspect he will recommend you scope all of these to be specific to the Output component.

Reply:
Included 'output-component' as prefix to the classes used in output view. It's now bound to only output view.